engine/components-api/src/main/java/com/cloud/storage/StorageManager.java

```diff
@@ -155,6 +155,15 @@
     ConfigKey<String> PreferredStoragePool = new ConfigKey<String>(String.class, "preferred.storage.pool", "Advanced", "",
             "The UUID of preferred storage pool for allocation.", true, ConfigKey.Scope.Account, null);
 
+    ConfigKey<Boolean> MountDisabledStoragePool = new ConfigKey<>(Boolean.class,
+            "mount.disabled.storage.pool",
+            "Storage",
+            "false",
+            "Mount disabled storage pool after node reboot",
+            true,
+            ConfigKey.Scope.Cluster,
```

Comment:
Shouldn't this scope be set to StoragePool? If not, please update the configuration description mentioning that it impacts all the storage pools in the cluster

Reply:
This should be zone wide as vm can be migrated across cluster. I will update the description